README.md

```diff
@@ -1,2 +1,67 @@
 # adapt-dev-utils
 Task runner utilities for developer tasks and automation
+
+## How to use
+When installed through npm, you'll run commands like this:
+`adapt-utils sb-clone`
+
+When working directly out of the adapt-dev-utils repository, you'll run commands like this:
+`./bin/run sb-clone`
+
+To show available commands:
+`adapt-utils` 
+
+To view instructions and options for a given command:
+`adapt-utils help sb-clone` 
+
+Example of running a command:
+`adapt-utils sb-clone --sourceName="OOD Giving" --targetName="OOD Giving Dev"`
+
+Note that if you omit required options, most commands have an interactive mode which will provide prompts for you to step through instead. This is typically the easiest approach, but in contexts where you need to automate the process you can bypass this by providing arugments for all requried parameters on the command line. 
+
+## Configuration
+
+Configuration variables can be set in a few different ways:
+
+1. Environment variables. 
+`GITHUB_TOKEN=YOUR_TOKEN`
+1. The config-set command.
+`adapt-utils config-set github_token "your_token"`
+1. Using an option in the CLI command.
+`adapt-utils netlify-cline --githubToken="yourtoken"`
```

Comment:
~netlify-cline~ => netlify-cl**o**ne

Reply:
Fixed! (Sorry - somehow I completely missed these PR comments while I was sidetracked on other tickets the last couple weeks)